test/mechanics/timeIntegration/RK4.cpp

```diff
@@ -0,0 +1,57 @@
+#include "BoostUnitTest.h"
+
+#include <iostream>
```

Comment:
iostream should not be needed in unit tests. You could use a [`BOOST_TEST_MESSAGE`](http://www.boost.org/doc/libs/1_51_0/libs/test/doc/html/utf/user-guide/test-output/BOOST_TEST_MESSAGE.html), if you really need to print something.